subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/ConsoleGradleBuildGroupedTaskFunctionalTest.groovy

```diff
@@ -76,8 +85,11 @@ class ConsoleGradleBuildGroupedTaskFunctionalTest extends AbstractConsoleGrouped
                 }
             }
             
+            task silence {
+            }
```

Comment:
The same goes here, create another fixture that specifically targets silent tasks.